src/apply.jl

```diff
@@ -298,33 +298,29 @@ end
 """Apply() function for an ITN in the Vidal Gauge. Hence the bond tensors are required.
 Gate does not necessarily need to be passed. Can supply an edge to do an identity update instead. Uses Simple Update procedure assuming gate is two-site"""
 function vidal_apply(
```

Comment:
Seems like this could now just be called `apply` since it dispatches on the `VidalITensorNetwork` type.

Reply:
Agreed.